src/Controllers/controller.ts

```diff
@@ -0,0 +1,23 @@
+import axios from "axios";
+
+async function getRovers() {
+    const response = await axios({
+        url: "https://api.nasa.gov/mars-photos/api/v1/rovers?sol=1000&api_key=qj9vwpFaaunBJQeJYnDtZjW1tW7ckM3gBZdQUh28",
+        method: "get",
+    });
+    return (response.data);
+};
+
+async function getRoverPhotos(rover_name: string, sol: string, camera_type: string, page: string) {

```

Comment:
you could add the response type (we should define interfaces for the objects we have)